App.js

```diff
@@ -271,22 +267,19 @@ export default function App() {
         <View
           style={
             (message === 'Game Complete') ? styles.itemInvisible : (
-            odd ? [styles.item, styles.markSpot1] : styles.item )
+            odd ? [styles.item, styles.markSpot1] : [styles.item, styles.itemBlue] )
           }
         >
         <TouchableHighlight onPress={() => pressOdd()}>
           <Text style={styles.itemText}>ODD</Text>
           </TouchableHighlight>
         </View>
-        <View 
-          style={
-            (message === 'Game Complete') ? styles.itemInvisible : styles.item
-          }
-        >
-        <TouchableHighlight onPress={() => pressRoll()}>
-          <Text style={styles.itemText}>ROLL</Text>
-          </TouchableHighlight>
-        </View>
+        <Button
+          onPress={() => pressRoll()}
+          title="ROLL"
+          color="blue"
+          disabled={(message === 'Game Complete') ? true : false}
```

Comment:
There has to be a better way than inquiring what message I last generated. Any ideas?

Reply:
Could you check if the player position is past or at the max?

Reply:
To set the message I check if newPosition >= filteredBoard.length. But these 2 props are not available down in this return statement. Should I be creating an endOfGame state prop to use?

Reply:
So I added a new state property to handle this.

Reply:
You should be able to just add a `const endOfGame = position >= board.length` right before you return your JSX.

Reply:
Nope....didn't work.